### PR TITLE
Implement commit

### PR DESCRIPTION
**Pull request by nrc**
PTAL @Connor1996

### CODE REVIEW

**Comment by nrc** [comment]
cc #38

File: proto/proto/kvrpcpb.proto [hunk 1]
```diff
@@ -16,6 +16,7 @@ message LockInfo {
     uint64 lock_version = 2;
     bytes key = 3;
     uint64 lock_ttl = 4;
+    Op lock_type = 5;
```
Comment (Connor1996) [inline proto/proto/kvrpcpb.proto]: why Op in `LockInfo`?

**Comment by AndreMouche** [comment]
@andylokandy @zyh-hust PTAL

File: proto/proto/kvrpcpb.proto (hunk 1 above)
Comment (nrc) [reply proto/proto/kvrpcpb.proto]: I think it is a mistake, sorry

File: kv/tikv/storage/commands/prewrite.go [hunk 2]
```diff
@@ -73,8 +74,9 @@ func (p *Prewrite) HandleError(err error) interface{} {
 func (p *Prewrite) prewriteMutation(txn *kvstore.MvccTxn, mut *kvrpcpb.Mutation) (*kvrpcpb.LockInfo, error) {
 	key := mut.Key
 	// Check for write conflicts.
-	if write, writeCommitTS, err := txn.SeekWrite(key); write != nil && err == nil {
+	if write, writeCommitTS, err := txn.SeekWrite(key, kvstore.TsMax); write != nil && err == nil {
 		if writeCommitTS >= *txn.StartTS {
+			fmt.Printf("commit TS %d\n", writeCommitTS)
```
Comment (Connor1996) [inline kv/tikv/storage/commands/prewrite.go]: please clean it

**Pull request merged**